clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -439,7 +439,8 @@ public ApiException build(String message) {
                 return new InvalidTxnStateException(message);
             }
         }),
-    INVALID_PID_MAPPING(49, "The PID mapping is invalid",
+    INVALID_PRODUCER_ID_MAPPING(49, "The producer attempted to use a producerId which is not currently assigned to " +
+            "its transactionalId",
```

Comment:
Super nit: do we really want to say `producerId` and `transactionalId` in the message? Or should it simply be `producer id` and `transactional id`?